test_requests_kerberos.py

```diff
@@ -416,6 +417,14 @@ def test_handle_response_500_mutual_auth_required_failure(self):
 
             self.assertFalse(clientStep_error.called)
 
+            # re-test with error response sanitizing disabled
+            auth = requests_kerberos.HTTPKerberosAuth(sanitize_mutual_error_response=False)
+            auth.context = {"www.example.org": "CTX"}
+
+            r = auth.handle_response(response_500)
+
+            self.assertNotIsInstance(r, requests_kerberos.kerberos_.SanitizedResponse)
```

Comment:
I guess use `assertFalse(isinstance(r, requests_kerberos.kerberos_.SanitizedResponse))` for Python 2.6 support.